lamindb/models/history.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+
+
+class History(models.Model):
+    """Stores the edit history for all LaminDB tables."""
+
+    seqno = models.AutoField(primary_key=True)
+    migration_history_id = models.BigIntegerField()
+    id = models.UUIDField()
+    table_name = models.CharField(max_length=100)
+    record_id = models.BigIntegerField()
+    record_data = models.JSONField(null=True)
+    event_type = models.PositiveSmallIntegerField()
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
The `auto_now_add` is likely not going to work because it operates on the Python level.

See here: https://github.com/laminlabs/lamindb/pull/2640/files#r2038202637

Reply:
Good point. We're making the trigger add this anyway, so we don't need that annotation in Django-land.